packages/server/test/migration/migrations-diff.test.ts

```diff
@@ -0,0 +1,70 @@
+import fs from 'fs'
+import path from 'path'
+
+import { compareDatabases } from './utils/diff'
+import { decrement } from './utils/semver-decrement'
+import { startMessagingServer } from './utils/server'
+
+const TIMEOUT = 30000
+
+describe('Migrations diff tests', () => {
+  const migrationFolder = path.join(__dirname, '../../src/migrations')
+  const migrationFiles = fs.readdirSync(migrationFolder)
+  const migrations = migrationFiles
+    .filter((f) => !isNaN(Number(f.charAt(0)))) // Only keep migration files
+    .map((f) => {
+      const version = f.split('-')[0]
+      return [f, version, decrement(version)]
+    })
+
+  describe.each(migrations)('%s', (migrationName, migrationVersion, previousVersion) => {
+    const afterMigrationDatabase = `after_mig_${migrationName}`
+    const beforeMigrationDatabase = `before_mig_${migrationName}`
+
+    test(
+      'Should run each migrations up to the migration we are testing',
+      async () => {
+        await startMessagingServer(
+          {
+            command: `yarn start migrate up --target ${previousVersion}`,
+            launchTimeout: TIMEOUT
```

Comment:
Why use `yarn start`? Was something not working with `yarn dev`?

Reply:
It's just faster